index.bs

```diff
@@ -2916,6 +2916,246 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method ### {#sctn-signal}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signal(optional PublicKeyCredentialSignalOptions options = {});
+    };
+
+    dictionary PublicKeyCredentialSignalOptions {
+        UnknownCredentialIdReport              unknownCredential;
+        CurrentCredentialsReport               currentCredentials;
+    };
+
+    dictionary UnknownCredentialIdReport {
+        required USVString                     rpId;
+        required Base64URLString               credentialId;
+    };
+
+    dictionary CurrentCredentialsReport {
+        required USVString                     rpId;
+        required Base64URLString               userId;
+        CurrentCredentialUserDetails           user;
```

Comment:
Also just a nit: Should this simply be a separate report type, since they can be combined, say CurrentUserDetailsReport? The name "CurrentCredentialsReport" is slightly too specific otherwise.

Reply:
I split this into two reports.